commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/arguments/MapArgument.java

```diff
@@ -23,33 +25,33 @@
 @SuppressWarnings("rawtypes")
 public class MapArgument<K, V> extends Argument<LinkedHashMap> implements GreedyArgument {
 
-	private final char delimiter;
-	private final Function<String, K> keyMapper;
-	private final Function<String, V> valueMapper;
+	private final String delimiter;
```

Comment:
We decided that the delimiter should be `char` only: [View this for reference](https://discord.com/channels/745416925924032523/1103635409336025179/1103643250482499604) (You might wanna read a little bit more than just this message)